dev/openchpl_data-model.sql

```diff
@@ -621,13 +621,13 @@ CREATE TABLE openchpl.test_functionality (
 	name varchar(1000),
 	certification_edition_id bigint NOT NULL,
 	practice_type_id bigint,
-	certification_criterion_id bigint,
+	certification_criterion_id_deleted bigint,
```

Comment:
I like this renaming

Reply:
Renaming it is fine in the `v-dev` or `v-next`, but in our `dev/data-model` file it should move directly to the final state. As long as that doesn't break our tests, I think our goal is to not have any of the OBE stuff in the `./dev/*` files